src/test/java/com/lob/client/test/CheckTest.java

```diff
@@ -48,6 +48,9 @@ private AddressResponse getAddress() throws Exception {
         return Iterables.get(client.getAddresses(1).get(), 0);
     }
 
+    // Date time 5 days in the future
+    DateTime fiveDaysFromNow = new DateTime().plusDays(5);
```

Comment:
We should not be adding `sendDate` to all tests. Instead, can you write a new test similar to `testCreateCheckWithFile()` where we test the minimal check requirement with the addition of sendDate. You should also declare `fiveDaysFromNow` inside that test instead of having it a global variable. Similarly, a new test should be created for the Postcards and Letter tests.